wg-batch/annual-report-2022.md

```diff
@@ -5,56 +5,51 @@
 1. What work did the WG do this year that should be highlighted?
    For example, artifacts, reports, white papers produced this year.
 
-   -
-   -
-   -
+   - We're gathering feedback from multiple users running various batch-related frameworks and projects on kubernetes, see [our agenda](https://docs.google.com/document/d/1XOeUN-K0aKmJJNq7H07r74n-mGgSFyiEDQ3ecwsGhec/edit) for both past and upcoming topics.
+   - Closely collaborated with SIG Apps, SIG Node and SIG Scheduling to improve batch workloads on kubernetes:
+      - [2307 - Job tracking without lingering Pods](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2307-job-tracking-without-lingering-pods)
+      - [3329 - Retriable and non-retriable Pod failures for Jobs](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/3329-retriable-and-non-retriable-failures)
+   - [Introduction to Batch WG during KubeCon EU 2022](https://www.youtube.com/watch?v=XeX2zBOykC4)
+   - Create [Kueue: A Kubernetes-native Job Queueing](https://www.youtube.com/watch?v=YwSZUdU3iRY) project and release [2 major versions in 2022](https://github.com/kubernetes-sigs/kueue/releases).
+   - Organize Batch Days during [KubeCon EU 2022](https://kubernetesbatchdayeu22.sched.com/) and [KubeCon NA 2022](https://kubernetesbatchdayna22.sched.com/)
 
 2. What initiatives are you working on that aren't being tracked in KEPs?
 
-   -
-   -
-   -
+   - Continuing work on [Kueue](https://github.com/kubernetes-sigs/kueue/releases).
+   - Continuing work on supporting specialized hardware vertical through [Node Feature Discovery](https://github.com/kubernetes-sigs/node-feature-discovery)
+     and [Scheduler plugins](https://github.com/kubernetes-sigs/scheduler-plugins).
```

Comment:
Thanks for incorporating my feedback. Just a minor rephrasing NIT:
```suggestion
   - Continuing work on Topology-aware Scheduling by enhancing [Node Feature Discovery](https://github.com/kubernetes-sigs/node-feature-discovery)
     and [Scheduler plugins](https://github.com/kubernetes-sigs/scheduler-plugins) to enable support for specialized hardware in Kubernetes.
```

Reply:
done